test/postman/inference_api_test_collection.json

```diff
@@ -1,17 +1,17 @@
 {
```

Comment:
Good, further templating for data has made it generic and simple + adding new test cases will be adding row in data file